rdma_srv/src/constants.rs

```diff
@@ -16,3 +16,7 @@ pub const EVENT_TYPE_SET: &str = "set";
 pub const EVENT_TYPE_DELETE: &str = "delete";
 
 pub const GRPC_SERVER_ADDRESS: &str = "http://[::1]:51051";
+
+pub const RDMA_SVC_CLIENT_ROLE_NORMAL:i32 = 1;
```

Comment:
remove this one too, use ClientRole enum defined earlier.

Reply:
Updated.